Два верстати-автомати виготовляють однакові вироби, які попадають на загальний конвеєр. Продуктивність першого автомата
в два рази більша, ніж продуктивність другого. Перший верстат-автомат виготовляє 60% виробів вищого гатунку, а другий - 84%.
а) Знайти ймовірність того, що навмання взятий з конвейєра виріб вищого гатунку.
б) Знайти ймовірність того, що він виготовлений на першому верстаті.

Вирішення:


а)  Сумарно обидва автомати виробляють 100% продукції. В залежності від продуктивності кожного, перший виробляє 2/3 від 
    повного об'єму, а другий 1/3. 
    Відсоткова частка виробів вищого гатунку у першого автомату дорувнює 60%, що складає (2/3) * (6/10) продукції.
    Відсоткова частка виробів вищого гатунку у другого автомату дорувнює 84%, що складає (1/3) * (84/100) продукцї.
    Сумарна частка виробів вищого гатунку складає:

In [6]:
P = (2/3) * (6/10) + (1/3) * (84/100)
print(round(P, 2)*100, "%")

68.0 %


Робимо висновок, що з вірогідністю 68%, навмання взята деталь с конвейєра буде вищого гатунку.

б) Знаходження вірогідності виготовлення довільно взятої деталі на першому верстатіє:

In [7]:
P = (2/3)
print(round(P, 2)*100, "%")

67.0 %


Вірогідність того, що деталь була виготовлена на першому верстаті дорівнює 67%